src/group/managed_group/mod.rs

```diff
@@ -459,6 +482,10 @@ impl<'a> ManagedGroup<'a> {
                                 plaintext.sender.sender,
                                 &indexed_members,
                             );
+                            // Extract and store the resumption secret for the current epoch
+                            let resumption_secret = self.group.epoch_secrets().resumption_secret();
+                            self.resumption_secret_store
+                                .add(self.group.context().epoch(), resumption_secret.clone());
```

Comment:
Do we really need to clone here? If we put it in a store, it might be nicer to make it an `Option` in `EpochSecrets` or just just return/derive it separately. I believe then we don't need to derive `clone` at all.

Reply:
I agree that we could take it out here in the context of `ManagedGroup`, but since it's provided by `MlsGroup`, it might be too restrictive there.

Reply:
In the end it doesn't matter too much, as the resumption secret is actually intended to be potentially used multiple times. I'm ok to leave it as is for now.